tests/external/test_report_downloader.py

```diff
@@ -63,3 +63,27 @@ def test_invalid_provider_type(self):
                                  report_source='hereiam',
                                  report_name='bestreport',
                                  provider_type='unknown')
+
+    @patch('masu.external.downloader.aws.aws_report_downloader.AWSReportDownloader.__init__', return_value=None)
+    def test_get_current_report(self, fake_downloader):
+        """Test get_current_report function."""
+        downloader = ReportDownloader(customer_name='customer name',
+                                      access_credential='mycred',
+                                      report_source='hereiam',
+                                      report_name='bestreport',
+                                      provider_type=AMAZON_WEB_SERVICES)
+        with patch.object(AWSReportDownloader, 'download_current_report', return_value=self.file_list):
+            files = downloader.get_current_report()
+            self.assertEqual(len(files), len(self.file_list))
+
+    @patch('masu.external.downloader.aws.aws_report_downloader.AWSReportDownloader.__init__', return_value=None)
+    def test_get_current_report_error(self, fake_downloader):
+        """Test get_current_report function with error."""
+        downloader = ReportDownloader(customer_name='customer name',
+                                      access_credential='mycred',
```

Comment:
Needs fake_arn

Reply:
Sorry. I must be missing something.  fake_arn?

Reply:
It's in `tests/external/downloader/aws/__init__.py`. It landed with my async tasks work.